test/storage/compression_test.cpp

```diff
@@ -0,0 +1,74 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// compression_test.cpp
+//
+// Identification: test/storage/compression_test.cpp
+//
+// Copyright (c) 2015-16, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#include "common/harness.h"
+
+#include "storage/data_table.h"
+
+#include "executor/testing_executor_util.h"
+#include "storage/tile_group.h"
+#include "storage/database.h"
+#include "storage/tile.h"
+#include "storage/tile_group_header.h"
+#include "storage/tuple.h"
+
+#include "concurrency/transaction_manager_factory.h"
+
+namespace peloton {
+namespace test {
+
+//===--------------------------------------------------------------------===//
+// Compression Tests
+//===--------------------------------------------------------------------===//
+
+std::unique_ptr<storage::DataTable> data_table_test_table;
```

Comment:
This `data_table_test_table` doesn't need to be declared as a global variable here. And the variable name can be simplified as `data_table`.